sapbtp-operator-charts/values.yaml

```diff
@@ -2,19 +2,24 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+kube_rbac_proxy:
+  cpu_limit: 50m
+  memory_limit: 256Mi 
+  cpu_req: 5m
+  memory_req: 32Mi
 manager:
-  memory_limit: 200Mi
-  cpu_limit: 250m  
-  req_memory_limit: 20Mi
-  req_cpu_limit: 100m
+  cpu_limit: 50m
```

Comment:
What is the reason for changing the values?

Reply:
As part of the kyverno security policies, there is a policy called require-requests-limits. I looked together with David Freidlin on our components and these are the values he thought would match the component use. 
In case these numbers don't align with what you would like to have here, I would like to consult with him again, and maybe we can find numbers we can agree on or something like that.